src/track/trackplaytimers.cpp

```diff
@@ -0,0 +1,35 @@
+#include "track/trackplaytimers.h"
+
+TrackTimers::TimerQt::TimerQt()
+{
+    connect(&m_Timer,SIGNAL(timeout()),
+            this,SIGNAL(timeout()));
```

Comment:
Qt tries hard to emit the timeout() signal as precise as possible. 
Unfortunately this interferes with the waveforms, also living in the main thread, which really need this exactness.
As workaround, we have a less exact GUI Tick timer that fires all 50 ms. 
It is [Master]", "guiTick50ms"